templates/views/manage.twig

```diff
@@ -51,6 +51,20 @@
               </Button>
             </div>
 
+            <br>
+    
+            <div id="onidInvite">
+              <form>
+                <label for="submitOnid">Please enter a space separated list of ONID ids</label>
```

Comment:
The `for` label should correspond to an id of an input. It should be `onid` here because you're using it on your input.